css/style.css

```diff
@@ -594,7 +603,7 @@ teaser .teaser-image{
 /*********** Media Query ************/
 /* Media Query **********************/
 
-@media(min-width:600px){
+@media(min-width:1024px){
   .breadcrumbs {
     grid-column:span 3;
```

Comment:
why is this required? It works also with grid-column: 1/4 which is already set before.